config/config-example.js

```diff
@@ -368,6 +368,11 @@ exports.logchallenges = false;
  */
 exports.loguserstats = 1000 * 60 * 10; // 10 minutes
 
+/**
+ * persistbattles - Whether or not to persist battles through server restarts.
+*/
+exports.persistbattles = true;
```

Comment:
I don’t know if this should be true by default in the example file. I can’t speak for anyone else, but I know I would have disabled this had it existed when I started doing testing on a local server because I often restart or kill the server without actually ending the battle I was testing with.